KPI/views/mobile_usage_2021.view.lkml

```diff
@@ -0,0 +1,145 @@
+view: mobile_usage_2021 {
+  derived_table: {
+    sql:
+    select
+        submission_date,
+        app_name,
+        sum(dau) as dau,
+        sum(wau) as wau,
+        sum(mau) as mau,
+        sum(cdou) as cdou
+    from
+        ${mobile_usage_fields.SQL_TABLE_NAME} AS mobile_usage_fields
+    where
+        {% condition mobile_usage_2021.campaign %} campaign {% endcondition %}
+        AND {% condition mobile_usage_2021.canonical_app_name %} canonical_app_name {% endcondition %}
+        AND {% condition mobile_usage_2021.channel %} channel {% endcondition %}
+        AND {% condition mobile_usage_2021.country %} country {% endcondition %}
+        AND {% condition mobile_usage_2021.country_name %} country_name {% endcondition %}
+        AND {% condition mobile_usage_2021.distribution_id %} distribution_id {% endcondition %}
+        AND {% condition mobile_usage_2021.id_bucket %} id_bucket {% endcondition %}
+        AND {% condition mobile_usage_2021.os %} os {% endcondition %}
+    group by 1,2 ;;
+  }
+
+  filter: campaign {
+    suggest_explore: mobile_usage_fields
+    suggest_dimension: mobile_usage_fields.campaign
+    type: string
+  }
+
+  filter: canonical_app_name {
+    suggest_explore: mobile_usage_fields
+    suggest_dimension: mobile_usage_fields.canonical_app_name
+    type: string
+  }
```

Comment:
This means we will have both a filter-only field `canonical_app_name`, and a dimension `app_name` in this explore. That may be confusing, we could drop the `app_name` dimension, make this a dimension, and keep the suggest; that will keep it as a filter in the above query but also let it be a dimension in the explore.